unblob/handlers/executable/elf.py

```diff
@@ -262,6 +264,9 @@ def calculate_chunk(self, file: File, start_offset: int) -> Optional[ElfChunk]:
             return None
         end_offset = self.get_end_offset(file, start_offset, header, endian)
 
+        if end_offset > SSIZE_MAX:
```

Comment:
i think it is legit to have elf files over 2GB in size, also to have ELF content in a firmware ending over the 2GB offset. I think the problem is caused by a problem in LIEF parsing the file, so maybe the root cause should be handled.